calicoctl/commands/ipam.go

```diff
@@ -29,9 +29,10 @@ func IPAM(args []string) error {
 	doc := constants.DatastoreIntro + `Usage:
   calicoctl ipam <command> [<args>...]
 
-    release      Release a Calico assigned IP address.
-    show         Show details of a Calico assigned IP address,
-                 or of overall IP usage.
+    release          Release a Calico assigned IP address.
+    show             Show details of a Calico assigned IP address,
+                     or of overall IP usage.
+    strictaffinity   true/false
```

Comment:
I think we should have a general `configure` subcommand here, in case we want to expose other configuration options here in the future. 

e.g. `calicoctl ipam configure --strictaffinity` rather than `calicoctl ipam strictaffinity` 

WDYT?

Reply:
Makes sense. It is better if in future as you mentioned we want to make something else configurable.